## 17manpage_sessionname_warning.dpatch by  <hesso@pool.math.tu-berlin.de>
##
## DP: Discourage the use of 'sessionname' (see #103771).

@DPATCH@
diff -Naur screen-4.0.3.orig/doc/screen.1 screen-4.0.3/doc/screen.1
--- screen-4.0.3.orig/doc/screen.1	2008-06-01 23:40:56.000000000 +0200
+++ screen-4.0.3/doc/screen.1	2008-06-01 23:44:56.000000000 +0200
@@ -2603,9 +2603,10 @@
 .PP
 Rename the current session. Note, that for \*Qscreen -list\*U the
 name shows up with the process-id prepended. If the argument \*Qname\*U
-is omitted, the name of this session is displayed. Caution: The $STY 
-environment variables still reflects the old name. This may result in 
-confusion. 
+is omitted, the name of this session is displayed. Caution: Among other
+problems, the $STY environment variable still reflects the old name.
+Use of this command is strongly discouraged. Use the \*Q-S\*U commandline
+option if you need this feature.
 The default is constructed from the tty and host names.
 .sp
 .ne 3
diff -Naur screen-4.0.3.orig/doc/screen.texinfo screen-4.0.3/doc/screen.texinfo
--- screen-4.0.3.orig/doc/screen.texinfo	2008-06-01 23:40:31.000000000 +0200
+++ screen-4.0.3/doc/screen.texinfo	2008-06-01 23:44:56.000000000 +0200
@@ -1757,9 +1757,11 @@
 Rename the current session. Note that for @code{screen -list} the name
 shows up with the process-id prepended. If the argument @var{name} is
 omitted, the name of this session is displayed.@*
-@emph{Caution}: The @code{$STY}
-environment variable still reflects the old name. This may result in
-confusion.  The default is constructed from the tty and host names.
+@emph{Caution}: Among other problems, the @code{$STY}
+environment variable still reflects the old name. Use of this command
+is strongly discouraged. Use the @code{-S} commandline option if you
+need this feature.
+The default is constructed from the tty and host names.
 @end deffn
 
 @node Suspend, Quit, Session Name, Session Management
